src/cljs/athens/views/athena.cljs

```diff
@@ -316,11 +325,11 @@
                                                           string (:block/string x)]]
                                                 (if (nil? x)
                                                   ^{:key i}
-                                                  [:div (use-style result-style {:on-click (fn [_]
-                                                                                             (let [uid (gen-block-uid)]
-                                                                                               (dispatch [:athena/toggle])
-                                                                                               (dispatch [:page/create query uid])
-                                                                                               (navigate-uid uid)))
+                                                  [:div (use-style result-style {:on-click #(let [uid (gen-block-uid)]
+                                                                                              (dispatch [:athena/toggle])
+                                                                                              (dispatch [:page/create query uid])
+                                                                                               ;; TODO(agentydragon): Open the new page in sidebar if Shift is pressed
```

Comment:
Is this TODO left on purpose?

Reply:
yes, this PR does not work for new pages. I might get around to fixing that but the PR is probably already useful like this.